tools/webpack/vendors.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const { join } = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+
+const copiedVendors = {
+	'inert-polyfill': [
+		'wicg-inert/dist/inert.js',
+		'wicg-inert/dist/inert.min.js',
+	],
+};
+const importedVendors = {
+	react: { import: 'react', global: 'React' },
+	'react-dom': { import: 'react-dom', global: 'ReactDOM' },
+	'react-jsx-runtime': {
+		import: 'react/jsx-runtime',
```

Comment:
Should there be another entry point for `jsx-runtime-dev`?

Reply:
I'm not sure that we need it, since we always externalize to the same variable name and `react/jsx-runtime` already supports "prod/dev"